webpack.dev.config.js

```diff
@@ -88,7 +88,7 @@ module.exports = merge(baseConfig, {
     new webpack.NamedModulesPlugin(),
 
     new webpack.EnvironmentPlugin({
-      NETWORK: process.env.NETWORK,
+      NETWORK: "ganache",
```

Comment:
Does this not break local builds against staging, etc, that are dev builds against non-ganache networks?

Reply:
no, that is the default value if process.env.NETWORK is not set, this is how you use EnvironmentPlugin